src/ProductlineApp.WebUI/ClientApp/src/components/atoms/buttons/cancelButton/CancelButton.tsx

```diff
@@ -1,10 +1,10 @@
 import { Link } from 'react-router-dom';
 import './css/cancelButton.css';
 
-function CancelButton({ pathTo, close }: { pathTo?: any; close?: any }) {
+function CancelButton({ pathTo, close, setShowButtons, showButtons }: { setShowButtons?: any, showButtons?: boolean; pathTo?: any; close?: any }) {
```

Comment:
staraj sie nie uzywac typu "any". Sa do dyspozycji wszystkie typy podstawowe jak string, number, boolean itd., a jak chodzi o funkcje, to mozna je zdefiniowac uzywajac wyrazenia np.:
 "() => void" gdzie z lewej sa arugmenty funkcji (w tym przypadku brak) a z prawej zwracany typ funkcji